onnxruntime/python/tools/pytorch_export_contrib_ops.py

```diff
@@ -14,9 +15,13 @@
     raise ModuleNotFoundError(
         "This module is only useful in combination with PyTorch. "
         "To install PyTorch see https://pytorch.org/.")
+import torch.onnx
 import torch.onnx.symbolic_helper as sym_help
 import torch.onnx.symbolic_registry as sym_registry
 
+if typing.TYPE_CHECKING:
```

Comment:
please add a comment explaining why this is only imported conditionally.

Reply:
Done. Import modules needed only for type checking.